kiwmi/desktop/view.c

```diff
@@ -369,9 +369,10 @@ static void
 view_child_commit_notify(struct wl_listener *listener, void *UNUSED(data))
 {
     struct kiwmi_view_child *child = wl_container_of(listener, child, commit);
-    if (child->mapped && child->parent_mapped) {
-        view_child_damage(child);
-    }
+    // TODO: when can it map and not affect .mapped and/or .parent_mapped?
```

Comment:
I'd say it can't, but well... I'll try to do some testing with this myself.

Reply:
It was just a quick note for myself, I didn’t check where the problem could lie – I was interested in the workaround primarily. It also shows that it may really be a bad idea to cache `parent_mapped`, but until I know the problem I can’t really say.

Anyway, I’ll be glad for any hint.

Reply:
Apparently fixed